attestation-agent/attestation-agent/Cargo.toml

```diff
@@ -21,12 +33,17 @@ thiserror.workspace = true
 tokio = { workspace = true, features = ["fs"] }
 toml.workspace = true
 tonic = { workspace = true, optional = true }
+ttrpc = { workspace = true, features = ["async"], optional = true }
 
 [dev-dependencies]
 tokio = { workspace = true, features = ["macros", "rt-multi-thread"] }
 
+[build-dependencies]
+tonic-build = { workspace = true, optional = true }
+ttrpc-codegen = { workspace = true, optional = true }
+
 [features]
-default = ["rust-crypto"]
+default = ["rust-crypto", "coco_as", "kbs"]
```

Comment:
Not sure `coco_as` should be a default feature given some of the uncertainty there.